src/panels/config/automation/ha-automation-editor.ts

```diff
@@ -492,6 +525,16 @@ export class HaAutomationEditor extends PreventUnsavedMixin(
     }
   }
 
+  public hassSubscribe() {
+    return [
+      subscribeEntityRegistry(this.hass.connection, (entries) => {
```

Comment:
Dont use subscribe, use the full entities context that is available for all config pages:

```ts
  @state()
  @consume({ context: fullEntitiesContext, subscribe: true })
  _entityReg!: EntityRegistryEntry[];
  ```

Reply:
cool thanks, this should be also changed for scenes then?

Reply:
And if you want just optimize is, and only update on the changes of 1 entry, do:

```ts
  @consume({ context: fullEntitiesContext, subscribe: true })
  @transform<EntityRegistryEntry[], EntityRegistryEntry>({
    transformer: (this, value)  =>
      value.find(
          ({ entity_id }) => entity_id === this._entityId
        );
  })